src/views/resources/form/products.html

```diff
@@ -138,6 +138,14 @@ <h4 class="i18n">
                     <input type="number" class="form-control" name="quantity"
                       min="0" max="9999999" step="any"/>
                   </div>
+                  <div class="form-group">
+                    <label class="i18n">
+                      <span data-lang="en_us">Availability</span>
+                      <span data-lang="pt_br">Disponibilidade</span>
+                    </label>
+                    <input type="number" class="form-control" name="production_time.days"
+                      min="0" max="9999999" step="any"/>
```

Comment:
```suggestion
                      min="0" max="99999"/>
```

Pela API o máximo pra esta propriedade é `99999`, e o `step="any"` é pra aceitar decimais, aqui é só inteiros então podemos deixar o step 1 (padrão).